### ISSUE TITLE
Plan: sensors created should also carry the name of the component.

### ISSUE THREAD

**Issue by jeroenterheerdt** [jeroenterheerdt/HAsmartirrigation#22]

**Comment by posixx** [comment]
agreed. i will wait for this to be implemented to prevent me from changing automations too often.

**Comment by posixx** [comment]
service is using instance name so now i have 2 services to call, look good! in order for the names getting to long i would suggest replacing "smart_irrigation" on the sensor names with the instance name
